src/nrf5_sdk15_platform/yield/ruuvi_nrf5_sdk15_yield.c

```diff
@@ -88,6 +88,13 @@ static void wakeup_handler (void * p_context)
     m_wakeup = true;
 }
 
+// Test if in interrupt mode
```

Comment:
This comment could be removed, function better detailed in .h file.

Reply:
Removed